src/ray/gcs/gcs_server/gcs_init_data.cc

```diff
@@ -0,0 +1,120 @@
+// Copyright 2017 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ray/gcs/gcs_server/gcs_init_data.h"
+
+namespace ray {
+namespace gcs {
+void GcsInitData::AsyncLoad(const EmptyCallback &on_done) {
+  // There are 5 kinds of table data need to be loaded.
```

Comment:
// There are 5 kinds of table data need to be loaded.
```suggestion
  // There are 6 kinds of table data need to be loaded.
```